recipe/meta.yaml

```diff
@@ -1,20 +1,27 @@
+{% set cupti_version= "11.8.0" %}
+
 package:
   name: cupti
-  version: 11.3.1
+  version: {{ cupti_version }}
 
 source:
-  path: .
+  url: https://developer.download.nvidia.com/compute/cuda/{{ cupti_version }}/local_installers/{{ runfile_name }}
+  sha256: 9223c4af3aebe4a7bbed9abd9b163b03a1b34b855fbc2b4a0d1b706ac09a5a16 
 build:
   number: 0
+  skip: True # [not (linux and x86_64)]
+  missing_dso_whitelist:
+    - '*'
 
 requirements:
   run:
-    cudatoolkit >=11.3,<11.4
+    # from https://docs.nvidia.com/cupti/r_main.html#r_compatibility_requirements
+    cudatoolkit >={{ cupti_version }}
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/libcupti.so           # [linux]
-    - test -f ${PREFIX}/lib/libcupti.so.11.3      # [linux]
+    - test -f ${PREFIX}/lib/libcupti.so
+    - test -f ${PREFIX}/lib/libcupti.so.{{ '.'.join(cupti_version.split('.')[0:2]) }}
 
 # CUDA Profiling Tools Interface (CUPTI) Library
 about:
```

Comment:
To satisfy SPDX license names I would rename the license as:
`license: LicenseRef-ProprietaryNvidia`
see an example https://github.com/AnacondaRecipes/intel_repack-feedstock/blob/5a271776e6cd6b3940e81521949f8b71d178fda0/recipe/meta.yaml#L480

Reply:
thanks